configure.ac

```diff
@@ -337,6 +337,12 @@ AC_ARG_ENABLE(spdlog,
         [Disable spdlog]))
 AM_CONDITIONAL(USE_SPDLOG, [test x$enable_spdlog != xno])
 
+AC_ARG_ENABLE(next-protocol-version-unsafe-for-production,
```

Comment:
should this be enabled in the ci script?

Reply:
Yes it should. I missed that. I will update shortly.

Reply:
Fixed.